docs/database-views/input4MIPs_source-id_CMIP6Plus.html

```diff
@@ -225,22 +225,6 @@ <h4>
       <td><a href='https://aims2.llnl.gov/search?project=input4MIPs&versionType=all&activeFacets=%7B%22source_id%22%3A%22MRI-JRA55-do-1-6-0%22%2C%22mip_era%22%3A%22CMIP6Plus%22%7D' target='_blank'>Published</a></td>
       <td>CMIP6Plus</td>
       <td>MRI-JRA55-do-1-6-0</td>
-      <td>True</td>
```

Comment:
Have we lost a dataset, again?

Reply:
No the metadata has been cleaned up so now there is only a version that has latest equal to "true" whereas previously there was both "True" and "true" in the globus index (I think there's a casting problem in the index, the value for latest is stored as string so both "True" and "true" are allowed whereas it should have been a boolean - I don't think it's easily fixed though so I've just not worried about it)